lib/stdlib/src/erl_stdlib_errors.erl

```diff
@@ -988,6 +993,8 @@ expand_error(bad_update_op) ->
     <<"not a valid update operation">>;
 expand_error(bitstring) ->
     <<"is a bitstring (expected a binary)">>;
+expand_error(calling_self) ->
+    <<"process sent a blocking request to itself">>;
```

Comment:
```suggestion
    <<"the device is not allowed to be the calling process">>;
```
Would this be more helpful to the user?

Reply:
Nice! Maybe “the device cannot be the current process”?